.pre-commit-config.yaml

```
@@ -35,7 +35,8 @@ repos:
     rev: 3.9.2
     hooks:
       - id: flake8
-        additional_dependencies: [flake8-bugbear]
+        additional_dependencies:
+          [flake8-bugbear, flake8-simplify, flake8-comprehensions]
```

Nit Pick: Alphabetical please.
```suggestion
          [flake8-bugbear, flake8-comprehensions, flake8-simplify]
```